packages/progress-bar/docs/ProgressBar.stories.tsx

```diff
@@ -1,96 +1,83 @@
 import React from "react"
+import { Story } from "@storybook/react"
 import { CATEGORIES } from "../../../storybook/constants"
 import { figmaEmbed } from "../../../storybook/helpers/figmaEmbed"
 import { ProgressBar } from "../index"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
 
 export default {
-  title: `${CATEGORIES.components}/ProgressBar`,
+  title: `${CATEGORIES.components}/Progress Bar`,
   component: ProgressBar,
   parameters: {
     docs: {
       description: {
         component: 'import { ProgressBar } from "@kaizen/progress-bar"',
       },
     },
-  },
-}
-
-export const DefaultStory = () => (
-  <ProgressBar
-    value={25}
-    max={100}
-    mood="positive"
-    isAnimating={true}
-    label="25%"
-  />
-)
-DefaultStory.story = {
-  name: "Positive (Kaizen Site Demo)",
-  parameters: {
     ...figmaEmbed(
       "https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%E2%9D%A4%EF%B8%8F-UI-Kit%3A-Heart?node-id=1929%3A20890"
     ),
   },
 }
 
-export const PositiveSubtext = () => (
+export const DefaultStory = () => (
   <ProgressBar
-    isAnimating={false}
     value={25}
     max={100}
     mood="positive"
-    label="Label"
-    subtext="Subtext"
-  />
-)
-PositiveSubtext.storyName = "Positive (with subtext)"
-PositiveSubtext.parameters = { chromatic: { disable: false } }
-
-export const PositiveAnimating = () => (
-  <ProgressBar
     isAnimating={true}
-    value={25}
-    max={25}
-    label="Label"
-    mood="positive"
-  />
-)
-PositiveAnimating.storyName = "Positive (isAnimating)"
-
-export const Informative = () => (
-  <ProgressBar
-    isAnimating={false}
-    value={25}
-    max={100}
-    mood="informative"
-    label="Label"
-  />
-)
-Informative.parameters = { chromatic: { disable: false } }
-
-export const Negative = () => (
-  <ProgressBar
-    isAnimating={false}
-    value={25}
-    max={100}
-    mood="negative"
-    label="Label"
+    label="25%"
   />
 )
-Negative.parameters = { chromatic: { disable: false } }
+DefaultStory.storyName = "Default (Kaizen Demo)"
 
-export const Cautionary = () => (
-  <ProgressBar
-    isAnimating={false}
-    value={25}
-    max={100}
-    mood="cautionary"
-    label="Label"
-  />
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => (
+  <StoryWrapper isReversed={isReversed}>
```

Comment:
Are you going to add a reversed sticker sheet? If not:

```suggestion
const StickerSheetTemplate: Story = () => (
  <StoryWrapper>
```

Reply:
I've added reversed sheet and creating card to create reversed variant

Reply:
We might not need a reversed variant of the component design-wise, but seeing how it looks on a reversed background is nice nonetheless 😄

Reply:
The labels (percentage and subtext)  are black so the reversed will have to make them white

Reply:
Oh! Yes that makes sense. Good pick up 👍🏻 We're reaping the benefits of having both sticker sheets already 😆